### PR TITLE
EZQMS-342: Add text editor configurable active highlighted node 

### PR DESCRIPTION
**Pull request by annano**
# Contribution checklist

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [ ] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [ ] - Does the translations are up to date?
* [ ] - Does it well tested?
* [ ] - Tested for Chrome.
* [ ] - Tested for Safari.
* [ ] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [ ] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [ ] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

A list of closed updated issues

### CODE REVIEW

**Review by aonnikov — DISMISSED** [review]

File: packages/text-editor/src/components/extension/nodeHighlight.ts [hunk 1]
```diff
@@ -237,7 +227,7 @@ const createDecorations = (
           range.to,
           mergeAttributes(
             attributes,
-            nodeUuid === storage.activeNodeUuid ? { class: 'text-editor-highlighted-node-selected' } : {}
+            options.isActiveNode?.(nodeUuid) === true ? { class: 'text-editor-highlighted-node-selected' } : {}
```
Comment (aonnikov) [inline packages/text-editor/src/components/extension/nodeHighlight.ts]: Why this code is not in `generateAttributes`?

File: packages/text-editor/src/components/extension/nodeHighlight.ts [hunk 2]
```diff
@@ -18,6 +18,7 @@ export function highlightUpdateCommand (): TextEditorCommand {
 
 export interface NodeHighlightExtensionOptions extends NodeUuidOptions {
   getNodeHighlightType: (uuid: string) => NodeHighlightType | undefined | null
+  isActiveNode?: (uuid: string) => boolean
```
Comment (aonnikov) [inline packages/text-editor/src/components/extension/nodeHighlight.ts]: Why do we need another callback for this? As long as all highlight is based on callback(s), can we return all required information from `getNodeHighlightType`?

Comment (annano) [reply packages/text-editor/src/components/extension/nodeHighlight.ts]: Updated

File: packages/text-editor/src/components/extension/nodeHighlight.ts (hunk 1 above)
Comment (annano) [reply packages/text-editor/src/components/extension/nodeHighlight.ts]: Updated

**Review by aonnikov — APPROVED** [review]

**Pull request merged**